src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -144,7 +152,7 @@ private void ReceiveReplicationBatches()
 
                             using (var msg = interruptibleRead.ParseToMemory(
                                 _replicationFromAnotherSource,
-                                "IncomingReplication/read-message",
+                                "IncomingReplicationStr/read-message",
```

Comment:
You wanted to probably use `$"{IncomingReplicationStr}/read-message"` here

Reply:
This is the original message I have not modified any of the internal logic.